products/angularjs.md

```diff
@@ -59,22 +59,22 @@ releases:
     latestReleaseDate: 2015-09-29
 
 -   releaseCycle: "1.2"
-    releaseDate: 2013-11-06
+    releaseDate: 2013-11-23
     eol: 2021-12-31
     latest: "1.2.32"
     latestReleaseDate: 2016-10-11
 
 -   releaseCycle: "1.1"
-    releaseDate: 2012-08-13
+    releaseDate: 2013-11-23
```

Comment:
A lot of dates are mistakenly set to 2013-11-23 [on npm.js](https://www.npmjs.com/package/angular?activeTab=versions) (such as 1.1.0 and 1.1.5). Unfortunately I don't see what to do except writing a custom script for this product.